proxy/proxy.go

```diff
@@ -105,6 +105,11 @@ func (s *Handler) doHijack(message *common.HttpMessage, key string, incomingMess
 			log.WithField("error", err).Errorf("Failed to read request %s", u.Host)
 		}
 		reader.Close()
+
+		for range incomingMessages {
+			// waiting for channel to close
+		}
+		conn.Close()
```

Comment:
Hm, this conn.close() succesfully makes the copy a few line below end, but it does so with an error. That causes us to log an error. Do you think we should tune down the log to info?